File: TicketFlowWeb/TicketFlowModel/Usuario/Genero.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TicketFlowModel.Usuario
{
    public class Genero
    {
        public int idGenero {  get; set; }
        public string nombre { get; set; }

        public Genero() { }

        public Genero(int idGenero, string nombre)
        {
            this.idGenero = idGenero;
            this.nombre = nombre;
        }
    }
}

File: TicketFlow/TicketFlowModels/src/main/java/pe/edu/pucp/ticketflow/usuario/model/Genero.java
package pe.edu.pucp.ticketflow.usuario.model;

public class Genero {
    public int idGenero;
    public String nombre;

    public Genero() {}

    public Genero(int idGenero, String nombre) {
        this.idGenero = idGenero;
        this.nombre = nombre;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public int getIdGenero() {
        return idGenero;
    }

    public void setIdGenero(int idGenero) {
        this.idGenero = idGenero;
    }
}
